part-spec/passives.json

```diff
@@ -0,0 +1,350 @@
+{
+    "$id": "https://github.com/chromeos/digital-datasheets/blob/main/part-spec/passives.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "passives",
+    "oneOf": [
+        {
+            "title": "resistor",
+            "type": "object",
+            "required": [
+                "partType",
+                "manufacturer",
+                "orderableMPN",
+                "sourceDatasheetID",
+                "digitalDatasheetID"
+            ],
```

Comment:
Let's discuss in the WG about adding the resistor value as a required property

Reply:
added